draft/spec/index.html

```diff
@@ -713,9 +713,14 @@ <h2>Inventory Digest</h2>
             the chosen digest algorithm for the object. An example might be <code>inventory.json.sha512</code>.
         </p>
         <p>
-            The digest sidecar file MUST contain the digest of the inventory file. This MUST follow the format
-            <code>DIGEST inventory.json</code>; that is, the digest of the inventory file, a single space, and the
-            string <code>inventory.json</code> which is the name of the inventory file in the same directory.
+            The digest sidecar file MUST contain the digest of the inventory file. This MUST follow the format:
+        </p>
+<pre>
+DIGEST inventory.json
+</pre>
+        <p>
+            One or more linear whitespace characters (spaces or tabs) must separate DIGEST from the string
```

Comment:
I think "linear whitespace" is rather confusingly defined in https://www.rfc-editor.org/std/std68.txt (in one place as space+tab, in another including CRLF; see https://stackoverflow.com/questions/21072713/what-exactly-is-the-linear-whitespace-lws-lwsp) and in https://www.ietf.org/rfc/rfc2616.txt (`LWS            = [CRLF] 1*( SP | HT )`) so I suggest removing the word "linear"